osu.Game/Overlays/Chat/ChatTextBar.cs

```diff
@@ -128,9 +128,8 @@ protected override void LoadComplete()
                 chattingTextContainer.FadeTo(showSearch ? 0 : 1);
                 searchIconContainer.FadeTo(showSearch ? 1 : 0);
 
-                // Clear search terms if any exist when switching back to chat mode
-                if (!showSearch)
-                    OnSearchTermsChanged?.Invoke(string.Empty);
+                if (showSearch)
+                    OnSearchTermsChanged?.Invoke(chatTextBox.Current.Value);
```

Comment:
Explanation for this change? Doesn't seem related to what this PR is claiming to be doing.

Reply:
I just wanted the behavior on search to be consistent even though it isn't an actual channel. If it's meant to be cleared every time, I'll revert.

Reply:
I don't know if it should or shouldn't clear but these kinds of weird flourishes being put into mostly unrelated PRs without prior discussion or explanation are tiring me out if I'm honest. I guess it's fine?